man/systemd.netdev.xml

```diff
@@ -1223,6 +1223,15 @@
           the tunnel.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>IgnoreDF=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true, enables IPv4 DF flag suppression on
+          the tunnel. Note that if <varname>IgnoreDF=</varname> is true,
+          <varname>DiscoverPathMTU=</varname> must be set to false.
+          Only applicable to GRE and GRETAP tunnels. Defaults to false.</para>
```

Comment:
Is the default value for DiscoverPathMTU false? The docs should really say that.

Maybe reword this as "Note that if IgnoreDF= is set to true, DiscoverPathMTU= cannot be set to false." (This way we don't imply that DiscoverPathMTU=false must be used.)

Please also expand DF. (See how DSCP is described below.)

Reply:
The default value of `DiscoverPathMTU` is true (https://github.com/systemd/systemd/blob/f2b98d59295c9e9d81a4e7f29122bb6e70e6b37b/src/network/netdev/tunnel.c#L1198). To set `IgnoreDF=` to true, `DiscoverPathMTU=` must be set to false.

I will expanded DF in the next push